.github/workflows/deploy.yml

```
@@ -0,0 +1,71 @@
+name: Deploy Workflow
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)